src/stackwalk.c

```diff
@@ -788,6 +788,30 @@ JL_DLLEXPORT void jl_print_backtrace(void) JL_NOTSAFEPOINT
     jlbacktrace();
 }
 
+JL_DLLEXPORT void jl_thread_print_all_task_backtraces(void)
+{
+    jl_safe_printf("Wassup!\n");
+    jl_task_t *ct = jl_current_task;
+    jl_ptls_t ptls = ct->ptls;
+    arraylist_t *live_tasks = &ptls->heap.live_tasks;
```

Comment:
Doesn't this only print tasks live on the current worker thread?

Reply:
yeah, but it's also lame, and not what we were originally hoping for...

It'd be nicer to have something that prints _all_ the live tasks, including tasks that are waiting on a lock somewhere. Am i correct that this would miss those, and would only print tasks that are on the thread's run queue? It's confusing though, since it _does_ seem to print sleeping tasks and sometimes _done_ tasks, which i don't understand.